src/search.cpp

```diff
@@ -211,18 +210,61 @@ void Search::clear() {
   Tablebases::init(Options["SyzygyPath"]); // Free mapped files
 }
 
+/// MainThread::perft() divide the perft search evenly through Threads
+void MainThread::perft() {
+
+  StateInfo st;
+  std::string posNow = rootPos.fen();
+  std::vector<PerftPos> newPerftRoot;
+  perftIdx.store(0);
```

Comment:
`std::memory_order_release`

Reply:
It failed on Travis

Reply:
Here: https://travis-ci.org/github/official-stockfish/Stockfish/builds/767300932

Reply:
hmm... I'll try `std::memory_order_release`